modules/unsupported/wfs-ng/pom.xml

```diff
@@ -206,6 +206,15 @@
       <version>2.0.0-2</version>
       <scope>test</scope>
 	</dependency>
+  <!-- enable this dependency to test saxon -->

```

Comment:
Maybe make it a profile, so that one can test it just adding an extra to the command line?